rpm/sapwmp.spec

```diff
@@ -71,12 +65,17 @@ Configuration and utilities for collecting SAP processes under control group to
 
 %install
 %make_install
-install -D -m 644 %{SOURCE1} %{buildroot}/%{_sysconfdir}/sapwmp.conf
-install -D -m 644 %{SOURCE2} %{buildroot}/%{_unitdir}/SAP.slice
-install -D -m 755 %{SOURCE3} %{buildroot}%{_prefix}/lib/supportconfig/plugins/sapwmp
-install -D -m 744 %{SOURCE4} %{buildroot}/%{_libexecdir}/sapwmp/wmp-sample-memory
-install -D -m 644 %{SOURCE5} %{buildroot}/%{_unitdir}/wmp-sample-memory.service
-install -D -m 644 %{SOURCE6} %{buildroot}/%{_unitdir}/wmp-sample-memory.timer
+%define wmpd %{_builddir}/%{name}-%{version}
```

Comment:
Interesting trick to get rid of %{SOURCEn}. IIUC, this also implies the `extract_file` service can be dropped from OBS `_service` file. (Not relevant to this repo, but just mentioning it.)

Reply:
Yes, `extract_file` no longer need in `_service` file any more. (Without you mentioning this, I didn't notice that part existed)

Will align the removed source and _service files in ibs after this pull request merged.